app/scripts/root.js

```diff
@@ -208,6 +209,16 @@ angular.module('openolitor-admin')
         }
       };
 
+      $rootScope.translateCalendar = function(){
```

Comment:
You are triggering the `$rootScope.translateCalendar` function on controller creation. Wouldn't it make sense to trigger it on on rootScope initialisation (you already do) and on `$rootScope.changeLang`?

Reply:
changed it and added some constant values to improve ux